mvlearn/decomposition/baseica.py

```diff
@@ -167,7 +167,7 @@ def fit(self, X, y=None):
                 self.individual_mixing_.append(lstq_solution.T)
         return self
 
-    def transform(self, X, y):
+    def transform(self, X, y=None):
```

Comment:
there are no y in transforms in sklearn.

it can be considered a bad decision but it's how it is